### DISCUSSION TITLE
Can I use saleor to start my business?

### DISCUSSION THREAD

**Discussion by a-for-alex** [saleor/saleor#7815] [Q&A] [answered]
Hi guys, I'm thinking to start my business using saleor as an ecommerce website. Is there a security issue currently? Also could anyone provide me a business website that already using a seleor?

**Comment by karolkielecki** [comment, marked as answer]
@a-for-alex Hi there! Saleor is an open-source platform--which means you can use it for your e-commerce project. Saleor is built with one of the most secure and trusted stacks: Python and Django. Saleor is backed by a 20-person, full-time team. We handle security for large volume clients and give the same security attention to Saleor OSS as to our largest clients.

Saleor has been designed for developers. You can find our official documentation [here](https://docs.saleor.io/docs/). We are also working on tutorials.

Please take a look at some of the projects powered by Saleor: 
- https://pfefferundfrost.de/ (using Next.js + Saleor)
- https://a-dam.com/

You can find more projects powered by Saleor on our website: [saleor.io](https://saleor.io/)

Good luck!

**Reply by a-for-alex** [reply]
It helps me a lot!! Thanks!
